.github/workflows/pull_request.yml

```diff
@@ -356,8 +356,8 @@ jobs:
             cmake \
                 -DCMAKE_CXX_COMPILER=$TOOLSET_CLANG_DIR/bin/clang++ \
                 -DCMAKE_C_COMPILER=$TOOLSET_CLANG_DIR/bin/clang \
-                -DCMAKE_BUILD_TYPE=Debug \
-                -DENABLE_ASAN=on \
+                -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+                 -DENABLE_ASAN=off \
```

Comment:
It's unacceptable.

Reply:
ok, but the build will fail with ASAN on now